tests/sentry/integrations/slack/test_notifications.py

```diff
@@ -50,6 +60,12 @@ def setUp(self):
             NotificationSettingOptionValues.ALWAYS,
             user=self.user,
         )
+        NotificationSetting.objects.update_settings(
```

Comment:
I suppose this doesn't hurt to have but there aren't any tests targeting issue alerts in this file

Reply:
`NewProcessingIssuesActivityNotification` reads from "issue alert" notification settings.

Reply:
hmmm then I'm confused how the test passed without it, maybe it was default before.